### ISSUE TITLE
snackbar-dialog: dialog should not be modal

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/evo-web#350]
### I verified there's no existing issue for this bug.

- [x] There are no existing issues

### Current behavior

Looking at the Marko storybook, it seems that aria-modal has been incorrectly set to true. It should be false.

https://opensource.ebay.com/evo-web/ebayui-core/?path=/story/dialogs-ebay-snackbar-dialog--default

See Skin example for reference:

https://opensource.ebay.com/evo-web/skin/component/snackbar-dialog

### Steps to reproduce

```bash
1.
2.
3.
...
```
